pr_agent/tools/pr_code_suggestions.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, pr_url: str, cli_mode=False):
                                           settings.pr_code_suggestions_prompt.user)
 
     async def suggest(self):
-        assert type(self.git_provider) == GithubProvider, "Only Github is supported for now"
+        assert type(self.git_provider) != BitbucketProvider, "Bitbucket is not supported for now"
 
         logging.info('Generating code suggestions for PR...')
         if settings.config.publish_review:
```

Comment:
**Suggestion:** The `suggest` method has an assertion to check if the git provider is not Bitbucket. It would be better to raise a more descriptive error message if the git provider is not supported.
```suggestion
    async def suggest(self):
        if type(self.git_provider) == BitbucketProvider:
            raise NotImplementedError("Bitbucket is not supported for now")

        logging.info('Generating code suggestions for PR...')
        if settings.config.publish_review:
```